redisinsight/ui/src/pages/databaseAnalysis/components/top-keys/TopKeys.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { EuiButton, EuiTitle } from '@elastic/eui'
+import { TableView } from 'uiSrc/pages/databaseAnalysis'
+import { Key } from 'apiSrc/modules/database-analysis/models/key'
+
+import Table from './Table'
+import Loader from '../table-loader'
+import styles from '../styles.module.scss'
+
+export interface Props {
+  topKeysLength: Key[]
+  topKeysMemory: Key[]
+  loading: boolean
+  delimiter?: string
+}
+
+const TopKeys = (props: Props) => {
+  const { topKeysLength, topKeysMemory, delimiter, loading } = props
+  const [tableView, setTableView] = useState<TableView>(TableView.MEMORY)
+
+  if (loading) {
+    return <Loader />
+  }
+
+  if ((!topKeysLength?.length) && (!topKeysMemory?.length)) {
+    return null
+  }
+
+  return (
+    <div className={styles.topNamespaceView}>
+      <EuiTitle className="section-title">
+        <h4 data-testid="top-keys-title">
+          {topKeysLength.length < 15 && topKeysMemory?.length < 15
```

Comment:
can these length be different?

Reply:
I mean topKeysLength and topKeysMemory

Reply:
i guess no, but it will be strange if we will check only one